src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-wed calendar--month-length-31">
```

Comment:
please check [this BEM cheatset](https://mate-academy.github.io/fe-program/css/typical-bem-mistakes)
Pay attention to how we name modifiers with values and fix it